Dockerfile.gpu

```diff
@@ -1,4 +1,4 @@
-ARG base=nvidia/cuda:11.6.2-cudnn8-runtime-ubuntu20.04
+ARG base=nvidia/cuda:11.3.0-cudnn8-runtime-ubuntu20.04
```

Comment:
My development machine's nvidia driver can not be upgraded, 11.6.2 can not be tested in my case, so I modified the cuda version. It would be nice to have an environment to test.